docs-src/cloud/operating-envelope-availability.md

```diff
@@ -0,0 +1,44 @@
+---
+id: operating-envelope-availability
+title: What is Temporal Cloud's SLA on Availability?
+sidebar_label: Availability
+description: Temporal Cloud provides 99.9% availability of the cloud service and a 99.9% guarantee against service errors.
+tags:
+  - temporal-cloud
+  - operations
+  - explanation
+---
+
+The Temporal Cloud service level agreement (SLA) promises 99.9% availability of the cloud service and a 99.9% guarantee against service errors.
```

Comment:
```suggestion
description: Temporal Cloud provides 99.99% availability of the cloud service and a 99.9% guarantee against service errors.
tags:
  - temporal-cloud
  - operations
  - explanation
---

Temporal Cloud provides 99.99% availability, and its service level agreement (SLA) has a 99.9% guarantee against service errors.
```

[website](https://temporal.io/cloud) now says 4 nines availability, but that's not part of the SLA (the SLA is the "guarantee" part on the website)